database/postgres/subscription.go

```diff
@@ -288,6 +288,10 @@ func (s *subscriptionRepo) FetchDeletedSubscriptions(ctx context.Context, projec
 }
 
 func (s *subscriptionRepo) LoadAllSubscriptionConfig(ctx context.Context, projectIDs []string, pageSize int64) ([]datastore.Subscription, error) {
+	if len(projectIDs) == 0 {
+		projectIDs = []string{""}
+	}
+
```

Comment:
projectIDs is a compulsory filter, if it's empty it should return empty subscriptions.

Reply:
But that's not how the query works. When the db is empty (no projects yet) the service doesn't start up

Reply:
It's because of [this issue](https://github.com/jmoiron/sqlx/issues/196); we use `IN (?)` in the query

Reply:
@jirevwe that's my point exactly, this empty filter now won't match anything in db, we should just do this:

```go
if len(projects) == 0 {
    return []datastore.Subscription{}, nil
}
```

Reply:
True true